### PR TITLE
Create com.adilhanney.super_nonogram.json

### PR DESCRIPTION
**Pull request by adil192**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by adil192** [comment]
bot, build com.adilhanney.super_nonogram

**Comment by adil192** [comment]
bot, build com.adilhanney.super_nonogram

**Comment by adil192** [comment]
bot, build com.adilhanney.super_nonogram

**Comment by adil192** [comment]
Tested working
image

File: com.adilhanney.super_nonogram.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+				"install -Dm644 com.adilhanney.super_nonogram.metainfo.xml -t /app/share/metainfo/",
+		
+				"install -Dm644 icon-16x16.png /app/share/icons/hicolor/16x16/apps/com.adilhanney.super_nonogram.png",
+				"install -Dm644 icon-24x24.png /app/share/icons/hicolor/24x24/apps/com.adilhanney.super_nonogram.png",
+				"install -Dm644 icon-32x32.png /app/share/icons/hicolor/32x32/apps/com.adilhanney.super_nonogram.png",
+				"install -Dm644 icon-48x48.png /app/share/icons/hicolor/48x48/apps/com.adilhanney.super_nonogram.png",
+				"install -Dm644 icon-64x64.png /app/share/icons/hicolor/64x64/apps/com.adilhanney.super_nonogram.png",
+				"install -Dm644 icon-128x128.png /app/share/icons/hicolor/128x128/apps/com.adilhanney.super_nonogram.png",
+				"install -Dm644 icon-256x256.png /app/share/icons/hicolor/256x256/apps/com.adilhanney.super_nonogram.png",
+				"install -Dm644 icon-512x512.png /app/share/icons/hicolor/512x512/apps/com.adilhanney.super_nonogram.png",
```
Comment (barthalion) [inline com.adilhanney.super_nonogram.json:44]: Unless you're using it internally, 512z512 is enough.

Comment (adil192) [reply com.adilhanney.super_nonogram.json:44]: In the [requirements](https://docs.flathub.org/docs/for-app-authors/requirements/#application-icons) it says
> Applications must provide application icons in at least 64×64px and 128×128px sizes. Application icons should either be included as scalable SVGs or high-resolution PNGs.

Does this need updating or would I need 64x64, 128x128, and 512x512?

(In https://github.com/flathub/flathub/pull/4420 I've just used a 512x512 icon: we'll see if the build succeeds)

Comment (hfiguiere) [reply com.adilhanney.super_nonogram.json:44]: > Unless you're using it internally, 512z512 is enough.

That mean any other size was superfluous.

**Pull request closed without merging**